moveit_ros/moveit_servo/include/moveit_servo/pose_tracking.h

```diff
@@ -187,7 +188,7 @@ class PoseTracking
   // Flag that a different thread has requested a stop.
   std::atomic<bool> stop_requested_;
 
-  double angular_error_;
```

Comment:
@Abishalini @tylerjw are these changes compatible with #320 ?

Reply:
I'll look into it, but in the meantime, I would actually prefer we didn't use this process for forward-porting changes to moveit_servo.  The issue is that moveit_servo's implementation and interfaces have diverged from MoveIt1 enough I'd like to avoid bringing in changes from MoveIt1 to servo without reviewing if they are converted to have ROS2 interfaces / best practices.

Reply:
Do you suggest we should cherry-pick changes in the future and just ignore servo or simply delete the changes while keeping the merged commits? That would probably be a lot of extra work... Also these changes look like they should still be applicable